### ISSUE TITLE
using ignoreTag feature

### ISSUE THREAD

**Issue by joelransom** [picklesdoc/pickles#449]
Is ignoreTag present in 2.15.0? I know it's kinda new, just looked today to see if such a thing existed. I couldn't get it to work adding something like --ignoreTag=@ignore to the end of my command line. I also couldn't find an example of usage and the doc page doesn't exist. :) 

My whole command line is:
.<path>\pickles.exe --documentation-format=dhtml --feature-directory=.\<path> --output-directory=.\documentation --link-results-file=<path>\TestResult.xml --test-results-format=nunit --IgnoreTag=@ignore

Thanks, 

Joel

**Comment by dirkrombauts** [comment]
Hi,

Actually the parameter is called `excludeTags` (plural) - we will support multiple tags in the future and I wanted the parameters to be forward compatible.

If you call pickles using the `--help` argument, you will see a list of all parameters.

Thanks for spotting that the documentation doesn't exist BTW. I'll look into that.

**Comment by joelransom** [comment]
Thanks, I got it to work. Somehow I missed the --help though I thought I'd tried that, good to know. 

Suggestion about the functionality for the future: I have a Feature, **UserSearch** that has one scenario in it. I added an @Ignore tag to the scenario, and used the excludeTags option to ignore it. 
In the resulting report, the scenario didn't show up, but the feature still appears in the index, and the page has a **UserSearch** heading and nothing else. 

Personally, I would have expected that page to not be included at all if there are no scenarios on it, because that was my goal with the 'ignore' in the first place. 

Thanks,

Joel
